app/DoctrineMigrations/Version20180201165909.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types = 1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Ramsey\Uuid\Uuid;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180201165909 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $sql = <<<'xENDx'
+INSERT INTO ls_def_licence (identifier, uri, title, licence_text, updated_at)
+    SELECT * FROM (SELECT :uuid, :uri, :title, :licence_text, NOW()) AS tmp
+    WHERE NOT EXISTS (
+            SELECT title FROM ls_def_licence WHERE title = :title
+        ) LIMIT 1;
+xENDx;
+        $insertLicence = $this->connection->prepare($sql);
+        $uuid = Uuid::uuid5(Uuid::NAMESPACE_URL, "data:text/x-ref;src=EXTERNAL_LICENCE");
```

Comment:
The text that the UUID is being generated from is quite generic, would probably be better to use the URL mentioned in the licence_text field to make it easy to add others if desired.